tests/force_push_test.rs

```diff
@@ -477,26 +481,33 @@ fn test_force_push_identical_wrong_previous_approval() {
         Ok(()),
     );
 
-    github.mock_get_statuses("some-user", "some-repo", "abcdef0999999", Ok(vec![]));
+    let before_hash = "abcdef0999999";
+    let after_hash = "1111abc9999999";
+
+    github.mock_get_statuses("some-user", "some-repo", before_hash, Ok(vec![]));
     github.mock_get_timeline(
         "some-user",
         "some-repo",
         32,
         Ok(vec![
+            github::TimelineEvent::new_review(
```

Comment:
realized this test was not testing what I thought it was. I messed up `found_dismiss` doing this refactor and this test didn't catch it. Made sure this time it did fail before moving `found_dismiss = true;` back to its proper spot.